src/net.h

```diff
@@ -939,7 +939,7 @@ class CConnman
     void SocketHandlerListening(const Sock::EventsPerSock& events_per_sock);
 
     void ThreadSocketHandler() EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex, !mutexMsgProc);
-    void ThreadDNSAddressSeed() EXCLUSIVE_LOCKS_REQUIRED(!m_addr_fetches_mutex, !m_nodes_mutex);
+    void ThreadAddressSeed() EXCLUSIVE_LOCKS_REQUIRED(!m_addr_fetches_mutex, !m_added_nodes_mutex, !m_nodes_mutex);
```

Comment:
fc1a3719b0cd733985b1e6d9234819823c3bfab8 It looks like this thread safety annotation added here can be correspondingly removed from `ThreadOpenConnections()`, as it no longer accesses `m_added_nodes`.

```diff
--- a/src/net.h
+++ b/src/net.h
@@ -888,7 +888,7 @@ private:
-    void ThreadOpenConnections(std::vector<std::string> connect) EXCLUSIVE_LOCKS_REQUIRED(!m_addr_fetches_mutex, !m_added_nodes_mutex, !m_nodes_mutex);
+    void ThreadOpenConnections(std::vector<std::string> connect) EXCLUSIVE_LOCKS_REQUIRED(!m_addr_fetches_mutex, !m_nodes_mutex);
```

Reply:
Done, thanks!